src/main/java/uk/gov/hmcts/reform/et/syaapi/helper/EmployeeObjectMapper.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.et.syaapi.helper;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.ecm.common.model.ccd.Et1CaseData;
+
+@Slf4j
+@Service
+public class EmployeeObjectMapper {
+
+    public Et1CaseData getEmploymentCaseData(String caseData) {
+        ObjectMapper mapper = new ObjectMapper();
+        Et1CaseData data = null;
+        try {
+            data = mapper.readValue(caseData, Et1CaseData.class);
+        } catch (JsonProcessingException e) {
+            log.error(e.getMessage());
```

Comment:
You need to log the entire exception, otherwise you end up loosing the cause.  E.g.
log.error("Failed to ....", e);

Reply:
code review comments incorporated.